PayPal.xcodeproj/project.pbxproj

```diff
@@ -570,31 +581,27 @@
 			path = Models;
 			sourceTree = "<group>";
 		};
-		62D3FB122C3DB4B30046563B /* VenmoPaymentsTests */ = {
+		459633142C46BD51002008EF /* Recovered References */ = {
```

Comment:
I'm guessing we can remove these `Recovered References`?

Reply:
Nice catch, updated: [f9f2df3](https://github.com/paypal/paypal-ios/pull/286/commits/f9f2df331910ef706ef23c98f22ac7d3df8b45f3)